index.bs

```diff
@@ -217,48 +227,86 @@ dictionary SetHTMLOptions {
 </pre>
 
 The {{Sanitizer}} configuration object encapsulates a filter configuration.
-The same config can be used with both safe or unsafe methods. The intent is
+The same configuration can be used with both safe or unsafe methods, where
```

Comment:
```suggestion
The same configuration can be used with both safe and unsafe methods, where
```
Perhaps we should add quotation marks around "safe"?

Reply:
Bikeshed. Sorry: "unsafe and non-unsafe is terrible". How about saying safe and linking to Security Considerations that starts with a bit more of an explicit "This is what we consider safe for our scenario."?

Reply:
I'm not sure anyone suggested unsafe and non-unsafe?

Reply:
I suggested & non-suggested that. Nevermind that bit.

I saw your criticism and agree with you that just saying safe without quotes or any extra context could be ambiguous. Putting it in quotes is not a lot of more context either

Considering that, I suggest putting it in quotes and "safe" linking the Security Considerations section would make more sense to me.

Reply:
Added a definition to "safe and unsafe" where we first use the terms, and linked there. Added scare quotes, too.